src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,358 @@
+    return out
+
+
+def ges_rente_vorauss_erwerbsm(
+    erwerbsgemindert: bool,
+    m_pflichtbeitrag: float,
+    ges_rente_wartezeit_5: bool,
+) -> bool:
+    """
+    Eligibility for Erwerbsminderungsrente (public disability insurance claim).
+
+    Legal reference: § 43 Abs. 1  SGB VI
+
+    Parameters
+    ----------
+    erwerbsgemindert
+        See basic input variable :ref:`erwerbsgemindert <erwerbsgemindert>.
+    m_pflichtbeitrag
+        See basic input variable :ref:`m_pflichtbeitrag <m_pflichtbeitrag>.
+    ges_rente_wartezeit_5
+        See :func:`ges_rente_wartezeit_5`.
+    Returns
+    -------
+    Eligibility for Erwerbsminderungsrente (public disability insurance claim) as a bool
+    """
+
+    anspruch_erwerbsm_rente = (
+        erwerbsgemindert and ges_rente_wartezeit_5 and m_pflichtbeitrag >= 36
+    )
+
+    return anspruch_erwerbsm_rente
+
+
+def entgeltp_west_erwerbsm_rente(
+    entgeltp_west: float,
+    entgeltp_ost: float,
+    entgeltp_zurechnungszeit: float,
+) -> float:
+    """Entgeltpunkte accumulated in western germany which Erwerbsminderungsrente
+    is based on (public disability insurance)
+    In the case of the public disability insurance,
+    pensioners are credited with additional earning points.
+    They receive their average earned income points for
+    each year between their age of retirement and the "zurechnungszeitsgrenze".
+
+    Parameters
+    ----------
+    entgeltp_west
+        See basic input variable :ref:`entgeltp_west <entgeltp_west>
+    entgeltp_ost
+        See basic input variable :ref:`entgeltp_ost <entgeltp_ost>
+    entgeltp_zurechnungszeit
+        See :func:`entgeltp_zurechnungszeit`.
+
+    Returns
+    -------
+    Final pension points for Erwerbsminderungsrente (public disability insurance)
+
+    """
+
+    anteil_entgeltp_west = entgeltp_west / (entgeltp_west + entgeltp_ost)
+
+    out = entgeltp_west + (entgeltp_zurechnungszeit * anteil_entgeltp_west)
+
+    return out
+
+
+def entgeltp_ost_erwerbsm_rente(
+    entgeltp_west: float,
+    entgeltp_ost: float,
+    entgeltp_zurechnungszeit: float,
+) -> float:
+    """Entgeltpunkte accumulated in eastern germany which Erwerbsminderungsrente
+    is based on (public disability insurance)
+    In the case of the public disability insurance,
+    pensioners are credited with additional earning points.
+    They receive their average earned income points for
+    each year between their age of retirement and the "zurechnungszeitsgrenze".
+
+    Parameters
+    ----------
+    entgeltp_west
+        See basic input variable :ref:`entgeltp_west <entgeltp_west>
+    entgeltp_ost
+        See basic input variable :ref:`entgeltp_ost <entgeltp_ost>
+    entgeltp_zurechnungszeit
+        See :func:`entgeltp_zurechnungszeit`.
+
+    Returns
+    -------
+    Final pension points for Erwerbsminderungsrente (public disability insurance)
+
+    """
+
+    anteil_entgeltp_ost = entgeltp_ost / (entgeltp_west + entgeltp_ost)
```

Comment:
If we need this more frequently it would be useful to make a (trivial) function out of it.